extensions/sql-database-projects/src/test/netCoreTool.test.ts

```diff
@@ -41,8 +44,33 @@ describe('NetCoreTool: Net core install popup tests', function (): void {
 			should(result).true('dotnet is either not present or in /usr/local/share by default');
 		}
 	});
-});
 
-export async function sleep(ms: number): Promise<{}> {
-	return new Promise(resolve => setTimeout(resolve, ms));
-}
+	it('Should run dotnet command', async function (): Promise<void> {
+		this.timeout(10000 * 10); // higher timeout for this test
+
+		const netcoreTool = new NetCoreTool();
+		const projfile = path.join(__dirname, 'baselines', 'simpleProject.sqlproj');
+		const dacpacfile = path.join(__dirname, 'baselines', 'bin', 'debug', 'simpleProject.dacpac');
+
+		const buildHelper = new BuildHelper();
+		await buildHelper.createBuildDirFolder();
+		const arg = buildHelper.constructBuildArguments(projfile, buildHelper.extensionBuildDirPath);
+
+		const options: DotNetCommandOptions = {
+			argument: arg
+		};
+
+		try {
+			await netcoreTool.runDotnetCommand(options);
+			should(fs.existsSync(dacpacfile)).equal(true); //if net core is present
+		}
+		catch (error) {
+			should(error.message).equal(NetCoreInstallationConfirmation); // if net ore is not present
```

Comment:
dotnet core being present should be two different tests.  Is dotnet core not being present an easily mockable case?

Reply:
its not testing dotnet core's presence but dacpac creation path. that cant depend on mock since it needs to use it